shared-bindings/protomatter/Protomatter.c

```diff
@@ -314,10 +314,107 @@ STATIC mp_obj_t protomatter_protomatter_refresh(mp_obj_t self_in) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(protomatter_protomatter_refresh_obj, protomatter_protomatter_refresh);
 
+//|   .. attribute:: width
```

Comment:
I think width and height should be on framebufdisplay instead or additionally because displayio.Display has them.

Reply:
Already there :)